evm/src/main/java/org/hyperledger/besu/evm/precompile/MainnetPrecompiledContracts.java

```diff
@@ -86,4 +88,11 @@ public static void populateForBLS12(
     registry.put(Address.BLS12_MAP_FP_TO_G1, new BLS12MapFpToG1PrecompiledContract());
     registry.put(Address.BLS12_MAP_FP2_TO_G2, new BLS12MapFp2ToG2PrecompiledContract());
   }
+
+  public static void populateForCancun(
+      final PrecompileContractRegistry registry, final GasCalculator gasCalculator) {
+    populateForIstanbul(registry, gasCalculator);
+    // TODO: allow override to be configured?
```

Comment:
Is this intended to be done in another PR?

Reply:
Yes, I haven't decided if we should allow user-specified trusted setups, am loosely in favor of it at some point, but isn't needed yet.